File: src/RecordsManagmentWeb/ClientApp/src/app/records-table/record.ts
import { RecordStatusEnum } from './record-status';

export class RecordDoc {
    id: number;
    fileName: string;
    status: RecordStatusEnum;
    inScope: boolean;
    inLog: boolean;
    requestPackageId: number;
    esRef?: number;
    claimNumber: string;
    bundleNumber: string;
    pageNumber: number;
    orderNumber: number;
    startDate: string;
    log: string;
    phase: string;
}

File: src/Models/Record.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Models
{
    public class Record
    {
        public int Id { get; set; }
        public int RequestId { get; set; }
        public string RequestNumber { get; set; }
        public string FileName { get; set; }
        public RecordStatusEnum Status { get; set; }
        public bool InScope { get; set; }
        public bool InLog { get; set; }
        public int? EsRef { get; set; }
        public string ClaimNumber { get; set; }
        public string BundleNumber { get; set; }
        public int PageNumber { get; set; }
        public int OrderNumber { get; set; }
        public DateTime StartDate { get; set; }
        public string Log { get; set; }
        public string Phase { get; set; }
        public bool Archived { get; set; }
        public bool Duplicated { get; set; }

        public virtual Request Request { get; set; }
        
    }
}
